### ISSUE TITLE
Accuracy stays at zero

### ISSUE THREAD

**Issue by nikisarv** [BShakhovsky/PolyphonicPianoTranscription#7]
Hello,

I'm trying to run your code but during training accuracy dixon does not increase. 
` 936/2212 [===========>..................] - ETA: 53s - loss: 0.0374 - Dixon: 0.0000e+00
  944/2212 [===========>..................] - ETA: 52s - loss: 0.0373 - Dixon: 0.0000e+00`

I also had a problem with categorical_crossentropy and after i saw your response in another issue i changed it to binary_crossentropy but now i get the results above. I tried it both with the whole dataset and just a small part of it and nothing changes.

**Comment by BShakhovsky** [comment]
Hi,
 
That is strange. I just tried running the template, and "Dixon" accuracy is decreasing at first, but starts increasing very soon. I waited for around 1000 steps as in your example, and loss is also around 0.037, but "Dixon" is already above 0.16 by that time.
No idea comes to my mind, what may be the reason in your case. Sorry.

**Comment by nikisarv** [comment]
Hello again and thank you for your reply!

The minSecs and maxSecs you are using for your results is the 5 and 20 right?

Also, I'm using an older version of your code (with tf-gpu 1.15) because when I've tried the newer version I had a memory leak during model.fit(). I'm trying to fix this now and run the newer one. Could you please tell me which keras version you are using along with tensorflow 2.4 & python 3.8.8?

Thank you in advance,
NIki

**Comment by BShakhovsky** [comment]
Yes, currently __*minSecs*__ and __*maxSecs*__ are __*5*__ and __*20*__. But I don't think, different values could cause zero accuracy.
My Keras version is 2.4.3.

**Comment by nikisarv** [comment]
Thanks again for your quick reply 

I will keep looking for the problem resulting in zero accuracy in my case.
